### DISCUSSION TITLE
How do I select multiple photos in mobile browser

### DISCUSSION THREAD

**Discussion by scottdk** [LycheeOrg/Lychee#2076] [Q&A] [answered]
I can use Ctrl-click to multi-select on the desktop, but I can find how to do that in a mobile browser?

Lychee 4.13.0 (docker) 

Android 14
OneUI 6. 0
Brave 1.61.109
Edge 120.0.2210.84

**Comment by ildyria** [comment, marked as answer]
Not possible at this time. To be implemented.

TBH, Lychee is not really thought as mobile focused for photo administration. If someone want to implement this, we will most likely accept the PR, the issue is that there isn't much dev in the team. Obviously we focus on adding what functionality we find the most useful for our usecases.

**Reply by ildyria** [reply]
Mobile multi-select should be implemented with a HOLD and probably tick box in top right of images (similar to how UI/UX works on phone).
